### PR TITLE
NEURON is no longer an extra

### PR DESCRIPTION
**Pull request by wizmer**
Due to NEURON being now available on PyPi, it no longer needs special handling for installation.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,17 +22,14 @@
...
     install_requires=[
+        'bluepyopt>=1.9.37',
         'click>=6.7',
-        'functools32>=3.2;python_version<"3.0"',
         'morphio>=2.3.4',
-        'numpy>=1.14',
         'neurom>=1.4.15',
+        'neuron>=7.8',
+        'numpy>=1.14',
+        'pandas>=1.0.3',
```
Comment (asanin-epfl) [inline setup.py:31]: Can we use a lower version of Pandas here? >= 0.25? IS it necessary to use that high version?

Comment (wizmer) [reply setup.py:31]: Hum, it's best to keep it a the latest major version. What is your underlying thought ?

Comment (asanin-epfl) [reply setup.py:31]: My concern is Spack. They don't have a couple of latest major versions.

Comment (wizmer) [reply setup.py:31]: Good point Aleksei.
It seems BlueBrain/Spack is still at 0.25 but Spack/Spack is at 1.0.3
https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/py-pandas/package.py

So I think we can bump pandas in the Spack BlueBrain fork whenever we will need to.

Comment (asanin-epfl) [reply setup.py:31]: I don't guarantee that it would be easy.

**Pull request merged**